src/Components/Navbar.css

```diff
@@ -0,0 +1,102 @@
+body {
```

Comment:
Denumirea se scrie cu litera mare doar la componenta care returneaza jsx. in cazu nostru <Navbar/>, iar css cu litere mici. foldurile tot